### ISSUE TITLE
Could not locate handlePlatformMessage error in Flutter Embedding <> Android on release variant

### ISSUE THREAD

**Issue by joshuadeguzman** [flutter/flutter#43626]
## What I am trying to do
I was trying to create a release build for my app with an embedded Flutter views. But then I continuously get this error every time I visit the part of the app the use Flutter embedding.

## The error I encountered
```
[ERROR:flutter/shell/platform/android/platform_view_android_jni.cc(660)] Could not locate handlePlatformMessage method
[FATAL:flutter/shell/platform/android/library_loader.cc(24)] Check failed: result.
```

## Steps to reproduce
Trying to run my app 
1. Build the Android app with Flutter in release mode
2. Visit the part of the app that calls `FlutterMain.startInitialization(activity.getApplicationContext());`

_App crashes..._

**Target Platform:**
Android

**Target OS version/browser:**
Android 9 / Pie

**Devices:**
Google Pixel XL

## Logs
```
$ pwd
$ /dir/app/flutter_code
$ flutter run -v
```
```
   +4 ms] Result: {type: FlutterViewList, views: [{type: FlutterView, id: _flutterView/0x7bd146f620, isolate: {type: @Isolate, fixedId: true, id: isolates/381191423, name: main.dart$main-381191423, number: 381191423}}]}
[        ] <- accept
[        ] Connected to _flutterView/0x7bd146f620.
[   +1 ms] 🔥  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
[        ] An Observatory debugger and profiler on Pixel XL is available at: http://127.0.0.1:62329/JEpiNf9YKus=/
[        ] For a more detailed help message, press "h". To detach, press "d"; to quit, press "q".
```

`$ flutter analyze`
```
Analyzing core...                                                       
No issues found! (ran in 2.8s)
```

`$ flutter doctor -v`
```
[✓] Flutter (Channel unknown, v1.7.8+hotfix.3, on Mac OS X 10.14.5 18F132, locale en-PH)
    • Flutter version 1.7.8+hotfix.3 at /Users/joshuadeguzman/Documents/Tools/flutter
    • Framework revision b712a172f9 (4 months ago), 2019-07-09 13:14:38 -0700
    • Engine revision 54ad777fd2
    • Dart version 2.4.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/joshuadeguzman/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = /Users/joshuadeguzman/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS (Xcode 10.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.3, Build version 10G8
    ✗ CocoaPods installed but not initialized.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To initialize CocoaPods, run:
        pod setup
      once to finalize CocoaPods' installation.

[✗] iOS tools - develop for iOS devices
    ✗ libimobiledevice and ideviceinstaller are not installed. To install with Brew, run:
        brew update
        brew install --HEAD usbmuxd
        brew link usbmuxd
        brew install --HEAD libimobiledevice
        brew install ideviceinstaller
    ✗ ios-deploy not installed. To install:
        brew install ios-deploy

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 36.1.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[!] IntelliJ IDEA Ultimate Edition (version 2019.2.3)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.39.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.5.1

[✓] Connected device (3 available)
    • Pixel XL • HT6C90202587 • android-arm64  • Android 10 (API 29)
    • macOS    • macOS        • darwin-x64     • Mac OS X 10.14.5 18F132
    • web      • web          • web-javascript • Google Chrome 77.0.3865.120 
```

**Comment by BondarenkoStas** [comment]
@joshuadeguzman Can you please paste the whole log (running with --verbose flag)? The feature is still in the preview, can you try it with current master channel?

**Comment by joshuadeguzman** [comment]
Hi @BondarenkoStas, thanks for the help. Closing this issue now, I encountered the issue because of a proguard configuration we have to apply for it to work with our CI. I'll re-open this if something comes up. Otherwise, updating to the latest version of Flutter is the key for this add2app issue/s.

Right now, we can't use the master branch as it will break a lot of our protobuf files (forced to use 0.13.15). But thanks for the suggestion!
